.github/features_schema.json

```
@@ -1,45 +1,63 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object",
-  "required": ["service", "name", "features"],
+  "required": ["service", "name", "aws_provider","features", "emulation_level"],
   "properties": {
     "service": {
       "type": "string",
-      "description": "Service's abbreviation in AWS"
+      "pattern": "^[A-Za-z0-9]+$",
+      "description": "Short service name"
     },
     "name": {
       "type": "string",
+      "pattern": "^[A-Z][^.\n]*$",
       "description": "The display name of the service from AWS"
     },
+    "aws_provider": {
+      "type": "object",
+      "required": ["api"],
+      "properties": {
+        "api": {
+          "type": "string",
+          "pattern": "^[a-z0-9-_]+$",
+          "description": "Name of AWS API provider or method name"
+        },
+        "name": {
+          "type": "string",
+          "default": "default",
+          "pattern": "^[a-z0-9-_]+$",
+          "description": "Name of AWS provider"
```

```suggestion
          "description": "Name of AWS provider implementation"
```

This way it might be more clear that this refers to our own implementations